config/config_defaults.go

```diff
@@ -738,6 +738,12 @@ const CGRATES_CFG_JSON = `
 	"listen": "127.0.0.1:3868",					// address where to listen for diameter requests <x.y.z.y/x1.y1.z1.y1:1234>
 	"listen_net": "tcp",						// transport type for diameter <tcp|sctp>
 	"dictionaries_path": "/usr/share/cgrates/diameter/dict/",	// path towards directory holding additional dictionaries to load
+	// "ce_applications": [						// list of applications in dictionaries wanted to be included in Capability-Exchange. Written as any of the four below
```

Comment:
Please remove them from defaults since they will wrongly point out towards fake defaults. To document the format you can add simple comments and separate config samples.